docs/guides/site_styles/index.md

```diff
@@ -5,3 +5,40 @@ has_children: true
 parent: Guides
 nav_order: 3
 ---
+
+Site Styles is where creators and developers can define colours, fonts and custom CSS.
```

Comment:
I would add explicitly that we're discussing the section of the platform

**My Site > Style**